torch/onnx/symbolic_opset14.py

```diff
@@ -40,6 +40,13 @@ def reshape(g, self, shape):
 
 @parse_args("v", "v", "v", "v", "v", "i", "f", "f", "i")
 def batch_norm(g, input, weight, bias, running_mean, running_var, training, momentum, eps, cudnn_enabled):
+
+    if torch.is_autocast_enabled() and \
```

Comment:
Why condition on autocast?

Reply:
To minimize this change surface as type check is already done when autocast is not enabled.

Reply:
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/templates/TensorMethods.cpp#L6 and https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/TensorCompare.cpp are a generic pieces for the tensor implementation, but other places such as https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Blas.cpp#L121, https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cuda/Blas.cpp#L381 also implement their own checks (there are other spots too).

Reply:
Thanks for explaining.